### PR TITLE
Add a corrections form

### PR DESCRIPTION
**Pull request by mfairchild365**
The corrections form somehow got lost, I found it.

### CODE REVIEW

File: www/templates/html/CorrectionForm.tpl.php [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
+        
         <input type="hidden" name="method" value="EMAIL" />
         <input type="hidden" name="initial_pagetitle" value="UNL Directory" />
-        
-        <input type="submit" value="Submit" />
+        <input type="hidden" name="initial_url" value="" />
     </form>
+    <p class="success hidden">Thank you for your correction.</p>
 </div>
+<!--
```
Comment (kabel) [inline www/templates/html/CorrectionForm.tpl.php]: No HTML comments please.

File: www/templates/html/Peoplefinder.tpl.php [hunk 2]
```diff
@@ -38,11 +38,15 @@
...
     //Don't wrap the home page, because we want it to use bands
     $page->maincontentarea = $savvy->render($context->output, $outputTemplate);
+    
 } else {
     //Wrap everything else
-    $page->maincontentarea = '<div class="wdn-band record-container"><div class="wdn-inner-wrapper wdn-inner-padding-sm">' . $savvy->render($context->output, $outputTemplate) . '</div></div>';
+    $page->maincontentarea = '<div class="wdn-band record-container"><div class="wdn-inner-wrapper wdn-inner-padding-sm">' . $savvy->render($context->output, $outputTemplate);
 }
 
+$page->maincontentarea .= '<div class="wdn-band"><section id="modal_edit_form" class="modal-overlay" role="dialog" aria-expanded="false">
```
Comment (kabel) [inline www/templates/html/Peoplefinder.tpl.php]: I'd prefer this static content to be in it's own template file.

File: www/templates/html/CorrectionForm.tpl.php (hunk 1 above)
Comment (mfairchild365) [inline www/templates/html/CorrectionForm.tpl.php]: D'oh

File: www/js/directory.js [hunk 3]
```diff
@@ -797,6 +821,39 @@ define([
...
+		$(correctionFormSelector + ' input[name="name"]').val(name);
+		$(correctionFormSelector + ' input[name="email"]').val(email);
+		$(correctionFormSelector + ' textarea[name="message"]').val('');
+		$(correctionFormSelector + ' input[name="initial_url"]').val($('.permalink', $vcard).attr('href'));
+
+		//Initialize states
+		$context.find('form').removeClass('hidden');
+		$context.find('.success').addClass('hidden');
+		
+		$form.on('submit', function(e) {
```
Comment (kabel) [inline www/js/directory.js]: It looks like this handler is going to get added every time this is called. I don't think that's what you want.

File: www/js/directory.js [hunk 4]
```diff
@@ -797,6 +821,39 @@ define([
...
+		var email =  idm.getEmailAddress() || '';
+		
+		//Initialize values
+		$(correctionFormSelector + ' input[name="name"]').val(name);
+		$(correctionFormSelector + ' input[name="email"]').val(email);
+		$(correctionFormSelector + ' textarea[name="message"]').val('');
+		$(correctionFormSelector + ' input[name="initial_url"]').val($('.permalink', $vcard).attr('href'));
+
+		//Initialize states
+		$context.find('form').removeClass('hidden');
```
Comment (kabel) [inline www/js/directory.js]: `$context.find('form') == $form` ? We always want to reduce the number of times we go to the selector engine.

File: www/js/directory.js [hunk 5]
```diff
@@ -797,6 +821,39 @@ define([
 			checkSticky();
 		});
 	};
+	
+	var launchCorrectionModal = function($target) {
+		var correctionFormSelector = '.correction-form';
+		var $vcard = $target.closest('.vcard');
+		var $context = $('.corrections-template');
+		var $form = $('.corrections-template form');
```
Comment (kabel) [inline www/js/directory.js]: Better to use the established $context: `$('form', $context)`

File: www/js/directory.js (hunk 4 above)
Comment (mfairchild365) [inline www/js/directory.js]: Good point, however `$form` is no longer defined.

Comment (kabel) [inline www/js/directory.js]: I'd put it back so you can use it as context for the above value inits. I don't think `correctionFormSelector` should be necessary.

**Pull request merged**